src/app.tsx

```diff
@@ -22,12 +21,13 @@ export default class App extends React.Component<{}, AppState> {
   async componentDidMount() {
     // Fetch data from database
     const response = await axios.get<AppState>("/data");
-    const [headers, ...data] = response.data.data;
+    const [headers, ...raw] = response.data.data;
     const updated = response.data.updated;
 
+    const practitioners = raw.map(entry => new Practitioner(entry));
```

Comment:
Love the class based approach you took! I'm pretty new to using Typescript so I appreciate this nudge in the right direction for full project adoption!